src/zarr/abc/store.py

```diff
@@ -49,6 +80,21 @@ def __init__(self, *args: Any, mode: AccessModeLiteral = "r", **kwargs: Any) ->
 
     @classmethod
     async def open(cls, *args: Any, **kwargs: Any) -> Self:
+        """
+        Create and open the store.
+
+        Parameters
+        ----------
+        *args : Any
+            Positional arguments to pass to the store constructor.
+        **kwargs : Any
+            Keyword arguments to pass to the store constructor.
+
+        Returns
+        -------
+        store
```

Comment:
```suggestion
        Store
```
If there's a single entry here, it's interpreted as the return type.